D/DrWatson/Compat.toml

```diff
@@ -20,3 +20,8 @@ FileIO = "1.0.6-1"
 ["0.1"]
 Requires = "0.5.2-*"
 julia = "1.0.0-1"
+
+["0.2"]
+FileIO = "1.0.6"
+Requires = "0.5.2"
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.